helm/cluster-gcp/Chart.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: v2
 name: cluster-gcp
 description: A helm chart for creating Cluster API clusters with the GCP infrastructure provider (CAPG).
 type: application
-version: [[ .Version ]]
+version: 1 # [[ .Version ]]
```

Comment:
I think we need `[[ .Version ]]` ?

Reply:
Yeah, my bad, I had to change it to be able to update deps.
Fixed now